doc/api.md

```diff
@@ -142,9 +169,9 @@ Returns:
 -   400 WRONG_SYNTAX if the body doesn't comply with the schema.
 -   500 SERVER ERROR if there was any error not contemplated above.
 
-##### GET /iot/services
+##### GET /iot/services or GET /iot/configGroups
 
-Retrieves service groups from the database. If the servicepath header has the wildcard expression, `/*`, all the
+Retrieves service/configuration groups from the database. If the servicepath header has the wildcard expression, `/*`, all the
```

Comment:
```suggestion
Retrieves configuration groups from the database. If the servicepath header has the wildcard expression, `/*`, all the
```

As I think the right name is "configuration groups" and "service" is a kind of legacy that we want to deprecate.

Reply:
Fixed in 424d59d